pom.xml

```diff
@@ -121,6 +115,16 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
+            <version>1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.1</version>
```

Comment:
The latest version is 1.1.4. If this is like the other repos, I think it can be `<scope>test</scope>`.

Reply:
Nice. Updated.